### PR TITLE
Refine on some sample code and fix a bug of ots writer.

### PR DESCRIPTION
**Pull request by zhouzf05**
1. Add sample of ots writer.
2. Set reserved throught to 0 of all the table created in sample code.
3. Fix bug: ots writer won't exit as a executor thread pool is not shut down inside.

### CODE REVIEW

File: src/test/java/examples/OTSWriterSample.java [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+        System.out.println("TotalRows in table: " + totalCount);
+    }
+
+    private static void createTable(OTSClient client, String tableName)
+            throws ServiceException, ClientException {
+        TableMeta tableMeta = new TableMeta(tableName);
+        tableMeta.addPrimaryKeyColumn(COLUMN_GID_NAME, PrimaryKeyType.INTEGER);
+        tableMeta.addPrimaryKeyColumn(COLUMN_UID_NAME, PrimaryKeyType.INTEGER);
+        // 将该表的读写CU都设置为100
```
Comment (whybert) [inline src/test/java/examples/OTSWriterSample.java]: 将该表的读写CU都设置为0

File: src/test/java/examples/OTSWriterSample.java [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    private static void scanTable(OTSClient client, String tableName) {
+        RangeIteratorParameter param = new RangeIteratorParameter(tableName);
+        RowPrimaryKey startKey = new RowPrimaryKey();
+        startKey.addPrimaryKeyColumn("uid", PrimaryKeyValue.INF_MIN);
+        startKey.addPrimaryKeyColumn("gid", PrimaryKeyValue.INF_MIN);
+
+        RowPrimaryKey endKey = new RowPrimaryKey();
+        endKey.addPrimaryKeyColumn("uid", PrimaryKeyValue.INF_MAX);
+        endKey.addPrimaryKeyColumn("gid", PrimaryKeyValue.INF_MAX);
+
```
Comment (qiyuewuyi) [inline src/test/java/examples/OTSWriterSample.java]: 这里的uid，gid可以是COLUMN_UID_NAME，COLUMN_UID_NAME。

File: src/test/java/examples/OTSWriterSample.java [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        }
+
+        @Override
+        public void run() {
+            int start = id * rowsCount;
+            for (int i = 0; i < rowsCount; i++) {
+                RowPrimaryKey primaryKey = new RowPrimaryKey();
+                primaryKey.addPrimaryKeyColumn("gid", PrimaryKeyValue.fromLong(start + i));
+                primaryKey.addPrimaryKeyColumn("uid", PrimaryKeyValue.fromLong(start + i));
+
```
Comment (qiyuewuyi) [inline src/test/java/examples/OTSWriterSample.java]: uid，gid同上

File: src/test/java/examples/OTSWriterSample.java [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+        try {
+            // 创建表
+            createTable(client, tableName);
+
+            writeRow(tableName);
+
+            scanTable(client, tableName);
+
+            deleteTable(client, tableName);
+
```
Comment (qiyuewuyi) [inline src/test/java/examples/OTSWriterSample.java]: 示例里面的deleteTable操作最好默认是注释起来的，除非用户明确要删除，否则尽量不要去删用户数据。

**Pull request merged**